package.json

```diff
@@ -36,7 +35,9 @@
     "enzyme-adapter-react-16": "^1.1.1",
     "enzyme-adapter-react-helper": "^1.2.3",
     "fork-ts-checker-webpack-plugin": "^0.4.1",
+    "husky": "^0.14.3",
     "jest": "^22.1.4",
+    "lint-staged": "^7.2.0",
```

Comment:
Shouldn't we add `prettier` as `devDependencies`.

Reply:
Do we need it? `prettier` is a devDependency for `lint-staged` and works for me out of the box (I dont have prettier installed globally as well)

Reply:
However, `jsonlint` doesn't work out of box although that too is a `devDependency` for `lint-staged`. it asks me to do `npm install jsonlint`. Not sure whats going on

Reply:
Looks like [prettier](https://github.com/okonet/lint-staged/blob/d52e074f66ae3c604413fca48f0ba6b999c2fcfb/package.json#L64) is available as dependency